src/tests/conformance/cmd/squelch.rs

```diff
@@ -101,15 +195,36 @@ async fn c009_TM_SQUELCH_cannot_squelch_peer_ledger_proposals() {
                 },
             ) = synth_node.recv_message().await
             {
-                if validator_pub_key == node_pub_key {
-                    break;
+                if distant_node_keys.contains(&node_pub_key) {
+                    panic!("It shouldn't be possible to receive proposing legders from squelched nodes.");
```

Comment:
this panic would not crash the test; it would only be a `JoinError` beneath the timeout one

Reply:
ah, I thought I saw a task being spawned there (in which case a panic in a task wouldn't crash the test) :+1: